src/modules/claims/claims.service.ts

```diff
@@ -1481,10 +1481,10 @@ export class ClaimsService {
   ): Promise<CredentialVerificationResult> {
     const { payload, eip191Jwt } = roleEIP191JWT;
     const errors: string[] = [];
-    const issuerDID = this._signerService.did;
+    const issuerDID = roleEIP191JWT.payload.iss;
     const { status: issuerVerified, error } =
       await this._issuerVerification.verifyIssuer(
-        issuerDID,
+        issuerDID as string,
```

Comment:
Similar to above, I think it's better to throw an error than pass a possibly undefined value into a function that expects a string
```suggestion
        issuerDID,
```

Reply:
Throwing an error now. vc.issuer could be string or IIssuer, but when I check for vc.issuer.id, I get a type error? Only vc.issuer as string is valid. 
export interface IIssuer {
    id: string;
    [x: string]: unknown;
}